web_src/css/modules/header.css

```diff
@@ -135,7 +135,9 @@ h4.ui.header .sub.header {
 }
 
 /* open dropdown menus to the left in right-attached headers */
-.ui.attached.header > .ui.right .ui.dropdown .menu {
+/* this method must be used instead of "left menu" because otherwise the menu will flash on page load. */
```

Comment:
```suggestion
/* this method must be used instead of "left menu" because otherwise the menu will flash on page load because of `display` CSS. */
```